ci/tests.py

```diff
@@ -110,8 +110,14 @@ def test_generate_report(self):
                 'details': {
                     'personnel_code': 'personnel_code',
                     'facility_name': 'facility_name',
+                    'default_addr_type': 'msisdn',
                     'role': 'role',
                     'state': 'state',
+                    'addresses': {
+                        'msisdn': {
+                            '+2341234567890': {}
+                        }
+                    }
                 }
             },
             status=200,
```

Comment:
@DevChima mostly follow along and ask question for stuff that strike you as odd or don't understand :)